guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/GitMetadataStore.java

```diff
@@ -37,6 +37,14 @@ void write( String name,
                 String origin );
 
     /**
+     * Writes the meta information about a repository.
+     * @param name The name of the repository in organizationalUnit/repositoryName format
```

Comment:
@porcelli the main problem here is this part is prepared to track forks. You can write a single metadata object, but you can't relate it with its origin.